packages/kube-client/lib/Client.js

```diff
@@ -88,11 +88,34 @@ class Client {
     return kubeconfig
   }
 
+  async getShootAdminKubeconfig ({ name, namespace }) {
```

Comment:
In this case I am not sure if `createShootAdminKubeconfig` would be better

Reply:
✅